app/models/application_search.rb

```diff
@@ -30,7 +30,7 @@ def satisfies_all(app)
   def satisfied?(app, key, value)
     case key
     when :assigned_user_id
-      app.current_assignment.user_id == value
+      assigned_user_clause(app, value)
```

Comment:
👍

Reply:
Yeah rubocop was complaining but I think it makes it a little easier to parse.